routes/main.js

```diff
@@ -10,6 +10,11 @@ router.route(ROUTES.index).get((req, res) => {
   res.render("home", { extraStyles });
 });
 
+router.route(ROUTES.error).get((req, res) => {
+  const extraStyles = '<link rel="stylesheet" href="/stylesheets/cards.css" />';
```

Comment:
I would recommend to create a file, maybe `styles.js` or something that exports these constants.